diagnostic.md

```diff
@@ -5,74 +5,89 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The purpose of a backend is to respond to user requests from the client. These
+responses usually are the result of accessing a database to save or modify
+stored data.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The model is used by the controller to fetch data.
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The controller communicates with the model.
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Serializers replace views but not sure what serializers are or why they replace views.
```

Comment:
Basically the idea is that we build our own client 'views' with HTML/Handlebars/Javascript etcetera.